fiddler/tests/test_fiddler.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Callable, Dict
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.stubs.aggregator import AggregatorStub
+from datadog_checks.dev.utils import get_metadata_metrics
+from datadog_checks.fiddler import FiddlerCheck
+
+
+def test_check(dd_run_check, aggregator, instance):
+    # type: (Callable[[AgentCheck, bool], None], AggregatorStub, Dict[str, Any]) -> None
+    check = FiddlerCheck('fiddler', {}, [instance])
+    dd_run_check(check)
+
+    aggregator.assert_metric('fiddler.accuracy')
+    #    aggregator.assert_metric('fiddler.prediction_drift')
```

Comment:
Can we please remove this commented out line if the metric isn't being asserted?
```suggestion
    #    aggregator.assert_metric('fiddler.prediction_drift')
```

Reply:
Done.